client/src/widgets/Licenses.tsx

```diff
@@ -124,3 +147,98 @@ export function DisplayLicenseItem({
     </ListItem>
   );
 }
+
+export function LicenseDrawerBox({ children }: { children: React.ReactNode }) {
```

Comment:
Maybe just `LicenseBox`? The name `Drawer` made me first think it was a drawer, until I later realized it was to be placed in a drawer.